src/CFBG.cpp

```diff
@@ -42,6 +45,16 @@ bool CFBG::IsEnableBalancedTeams()
     return _IsEnableBalancedTeams;
 }
 
+bool CFBG::IsEnableEvenTeams()
```

Comment:
`IsEvenTeamsEnabled`

Reply:
it's to be coherent with the others config variables